analytics/settings.toml

```diff
@@ -1,4 +1,4 @@
 POSTGRES_NAME = "app"
-POSTGRES_HOST = "0.0.0.0"
+POSTGRES_HOST = "grants-analytics-db"
 POSTGRES_USER = "app"
 POSTGRES_PORT = 5432
```

Comment:
I believe you would need a password here as well, can you confirm?

Reply:
Theres a `secrets.toml` that's in the `gitignore` that has the password